### PR TITLE
physlr_remove_bridges_graph

### PR DESCRIPTION
**Pull request by afshinfard**
takes the overlap graph, removes the bridges in its MST(s) iteratively and output the bridge-removed overlap graph (not the backbone graph).
Purpose: troubleshooting the current missassembly detection method and developing a new method of correction.

### CODE REVIEW

**Comment by afshinfard** [comment]
@lcoombe I am still not sure if I should PR to master or not, the main purpose is troubleshooting (that may take some time) but like some other steps it take a long time to do this single step of removing bridges and I see it as a good checkpoint.

File: data/Makefile [hunk 1]
```diff
@@ -696,6 +696,10 @@ minsize=2000
 %.bic.tsv: %.tsv
 	$(python) $(bin)/physlr biconnected-components $< >$@
 
+# Determine the bridge-removed graph from the overlap TSV.
+%.bridge-removed.tsv: %.tsv
+	$(time) $(python) $(bin)/physlr bridge-removed --prune-branches=10 --prune-bridges=10 -s0 $< >$@
```
Comment (lcoombe) [inline data/Makefile]:
```suggestion
	$(time) $(python) $(bin)/physlr bridge-removal --prune-branches=10 --prune-bridges=10 -s0 $< >$@
```
I know this is a small suggestion -- I just wonder if that is a slightly more clear name

Comment (afshinfard) [reply data/Makefile]: yep I agree, especially as the `-` that grammatically (if am right) is needed in between is now hidden behind the common dash we use.

Comment (afshinfard) [reply data/Makefile]: am, just a small concern: 
Other functions are named with the type of data they output. For example: `physlr backbone`, `physlr backbone-graph`, `physlr mst`. and in this example we have a bridge-removed graph and that is why I used this naming... while bridge-removal is the action (?). I myself am still feel bad with bridge-removed but do you have alternatives that match name of other functions?
update: we do have functions like `physlr count-molecules`. so `physlr bridge-removal` or `physlr remove-bridges`?

Comment (lcoombe) [reply data/Makefile]: Yeah I like `remove-bridges`

**Comment by afshinfard** [comment]
and I had a test run to see if the function works. I simply ran the backbone stage on the output of this new checkpoint, and the backbone stage should remove no more bridges. and it did not.

**Pull request merged**